CONTRIBUTING.md

```diff
@@ -0,0 +1,21 @@
+## Contribution 방법
+
+0.  https://github.com/DE-labtory/koa 을 `fork`하기
+1.  작업 할 내용 이슈에 등록하기 or 이슈에 이미 등록된 작업 선택 (작업중인 이슈에 댓글로 표시)
+2.  이슈로 등록된 작업 수행
+3.  수행한 작업의 문서, 단위 테스트 필수로 작성하기
+4.  DE-labtory/koa 폴더에서 `go test -v -p 1 ./... `로 테스트 수행, 
+5.  goimports로 코드 포맷팅 ```goimports -w ./```
```

Comment:
we need parallel option?

Reply:
I agree with zeroFruit

Reply:
Oh, i just copied it from swim... thanks!